apps/nextjs-example/src/components/ui/use-toast.ts

```diff
@@ -155,7 +155,7 @@ function toast({ ...props }: Toast) {
       ...props,
       id,
       open: true,
-      onOpenChange: (open) => {
+      onOpenChange: (open: any) => {
```

Comment:
Why this `any` annotation?

Reply:
it started to complain on not having a type and I couldn't build, so just quickly added any